examples/ads.amp.html

```diff
@@ -1024,14 +1025,24 @@ <h2 id="purch">Purch 300x250</h2>
     <div fallback></div>
   </amp-ad>
 
+  <h2 id="capirs">Rambler&Co</h2>
+  <amp-ad width="500" height="250"
```

Comment:
the slot loads an ad of height 90. do you want to tune the dimension to make it looks nicer?